compiler/semantics/ir_visitor.py

```diff
@@ -651,6 +651,14 @@ def visitMix(self, ctx: BSParser.MixContext):
         else:
             time = (10, BSTime.SECOND)
 
+        if ctx.usein():
+            usein_time_temp = self.visitUsein(ctx.usein())
+            usein_time = (usein_time_temp['quantity'], usein_time_temp['units'])
+            #self.visit(ctx.usein())
```

Comment:
In general, commented out code should be removed.
The exception is when there is a known bug and you have written bandaid code to fix what otherwise should work.  Please remove old/commented out code throughout.